ofrak_core/test_ofrak/components/test_extfs_component.py

```diff
@@ -16,6 +16,9 @@
 import test_ofrak.components
 
 
+pytestmark = pytest.mark.skipif_missing_deps([ExtUnpacker])
```

Comment:
Could be me not understanding something pytest is implicitly doing, but why declare this variable here? I assumed the mark was a decorator, but I don't see this `pytestmark` variable being used to decorate any functions

Reply:
Same comment applies to the creation of a similar variable in the tar tests

Reply:
This is the pytest way to skip an entire test file/module

https://docs.pytest.org/en/stable/example/markers.html#marking-whole-classes-or-modules